ISSUE_TRIAGE.md

```diff
@@ -8,11 +8,11 @@ The core maintainers of the Grafana project are responsible for categorizing all
 
 Triage helps ensure issues resolve quickly by:
 
-- Ensuring the issue's intent and purpose is conveyed precisely. This is necessary because it can be difficult for an issue to explain how an end user experiences a problem and what actions they took.
-- Giving a contributor the information they need before they commit to resolving an issue.
+- Making sure the issues include a detailed description, error logs (if applicable), and the necessary steps to reproduce them before committing to a resolution.
```

Comment:
I think we may want to leave it as it is in the previous version tbh. I see you're trying to summarise the couple of lines but I think it was clearer earlier. Some thoughts:

- In your version it looks like the person in triage is the one committing to resolving the issue, that's the role of the contributor.
- You start to mention the necessary info in the issue but it's not complete, I think the previous version covers it with "intent and purpose", without having to enumerate the specific info that's needed.

Reading the intro, I think we can skip any changes here and move to the next sections tbh. I think the info here is clear and hasn't changed.